README.md

```diff
@@ -15,14 +15,9 @@ with additional meta-data. This is the version for Android.
 
 Kiwix Android is written in [Kotlin](https://kotlinlang.org/) (with a few old
 pieces in Java).
-
-[![Build Status](https://github.com/kiwix/kiwix-android/workflows/CI/badge.svg?query=branch%3Adevelop+workflow%3ANightly)](https://github.com/kiwix/kiwix-android/actions?query=workflow%3ANightly+branch%3Adevelop)
-[![codecov](https://codecov.io/gh/kiwix/kiwix-android/branch/develop/graph/badge.svg)](https://codecov.io/gh/kiwix/kiwix-android)
-[![CodeFactor](https://www.codefactor.io/repository/github/kiwix/kiwix-android/badge)](https://www.codefactor.io/repository/github/kiwix/kiwix-android)
-[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
-[![Public Chat](https://img.shields.io/badge/public-chat-green)](https://chat.kiwix.org)
-[![Slack](https://img.shields.io/badge/Slack-chat-E01E5A)](https://kiwixoffline.slack.com)
-
+|CI-[`develop`](https://github.com/kiwix/kiwix-android/tree/develop)|Code Coverage|Codefactor|License|IRC|Slack|Nightly Builds|
+|--|--|--|--|--|--|--|
+|[![Build Status](https://github.com/kiwix/kiwix-android/workflows/CI/badge.svg?query=branch%3Adevelop+workflow%3ANightly)](https://github.com/kiwix/kiwix-android/actions?query=workflow%3ANightly+branch%3Adevelop)|[![codecov](https://codecov.io/gh/kiwix/kiwix-android/branch/develop/graph/badge.svg)](https://codecov.io/gh/kiwix/kiwix-android)|[![CodeFactor](https://www.codefactor.io/repository/github/kiwix/kiwix-android/badge)](https://www.codefactor.io/repository/github/kiwix/kiwix-android)|[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)|[![Public Chat](https://img.shields.io/badge/public-chat-green)](https://chat.kiwix.org)|[![Slack](https://img.shields.io/badge/Slack-chat-E01E5A)](https://kiwixoffline.slack.com)|[![Nightly](https://github.com/kiwix/kiwix-android/actions/workflows/nightly.yml/badge.svg)](https://github.com/kiwix/kiwix-android/actions/workflows/nightly.yml)|
```

Comment:
This has been broken

Reply:
It fetches latest nightly status on develop, since it failed last time that's why it shows `failed` here, it'll be green once we get it building green on dev. It is dynamic

Reply:
OK, can ou please then just add the nigky badge? We don’t need the titles IMO.

Reply:
I believe readme should be beginner friendly

Reply:
Then please remove everything related to the badge and let discuss this in the ticket you have open on this topic.